pentagon/pentagon.py

```diff
@@ -312,24 +312,19 @@ def configure_default_project(self):
 
 
 class GCPPentagonProject(PentagonProject):
-    local_defaults = {
-        'working_cluster_name': 'working',
-        'production_cluster_name': 'production',
-    }
-
     def __init__(self, name, data={}):
         # Build translated data for inventory input
-        self._gcp_inventory_context = self._build_inv_params(data)
+        self._gcp_inventory_context = self._build_inv_params(name, data)
         # Add the project_name since it isn't passed in from click
         self._gcp_inventory_context['project_name'] = name
-        self._gcp_inventory_context['name'] = self._gcp_inventory_context['project']
+        self._gcp_inventory_context['project'] = self._gcp_inventory_context['gcp_project']
+        self._gcp_inventory_context['name'] = name
         super(GCPPentagonProject, self).__init__(name, data)
 
     @staticmethod
-    def _build_inv_params(input_context):
+    def _build_inv_params(name, input_context):
```

Comment:
Is `name` used?

Reply:
hmmm i guess not - it was going to be but there is a _lot_ of confusion on what variables get used where. It took a lot of trial and error to get `project_name`, `name` and `project` keys to be the "right" values. It's still unclear to me their uses downstream due to the way variables propagate. This `_build_inv_params` may need `name` in the future if someone discovers that i did this wrong. I can remove it and publish v3.0.1 if you'd like, but truly this is a byproduct of some confusion around variable names and keys.

Reply:
Agreed, they are poorly named and somewhat confusing.  Ther is still work to be done and we can roll that into #174 or #176